src/microbe_stage/PlacedOrganelle.cs

```diff
@@ -518,6 +547,18 @@ private void FinishAttachToMicrobe()
 
         MakeCollisionShapes(ParentMicrobe!.Colony?.Master ?? ParentMicrobe);
 
+        StoredEnzymes = new Dictionary<Enzyme, int>();
```

Comment:
As each organelle has this initialized anyway, I'd make the property non-nullable and add a default "= new()" to it.

Reply:
Unless Github is showing me an old diff, this is still in this file: `public Dictionary<Enzyme, int>? StoredEnzymes`

Reply:
Yes, I still haven't pushed my commits.

Reply:
Ah, that would certainly explain things. I'll just then only finish my review for the files I haven't looked at at all yet.